Detectors/MUON/MCH/Raw/Decoder/include/MCHRawDecoder/DataDecoder.h

```diff
@@ -91,6 +91,7 @@ class DataDecoder
   };
 
   struct SampaTimeFrameStart {
+    SampaTimeFrameStart() = default;
```

Comment:
I missing that previously but shouldn't mBunchCrossing (at line 97) be a uint32_t instead of a int32_t (just for consistency as it probably does not matters since it is a 20 bits max int anyway).

Reply:
Yes, you are right.